manifests/bind_exporter.pp

```diff
@@ -0,0 +1,142 @@
+# Class: prometheus::bind_exporter
+#
+# This module manages prometheus bind_exporter
+#
+# Parameters:
```

Comment:
can you provide the documentation in puppet-strings format? (like it happended in https://github.com/voxpupuli/puppet-prometheus/blob/master/manifests/unbound_exporter.pp )

Reply:
done